docs/postgres.md

```diff
@@ -63,79 +50,6 @@ You may need to enable password authentication so `synapse_user` can
 connect to the database. See
 <https://www.postgresql.org/docs/current/auth-pg-hba-conf.html>.
 
-If you get an error along the lines of `FATAL:  Ident authentication failed for
```

Comment:
I wanted to move this stuff out of the happy path, so it's now in "troubleshooting" below.